sdk/boms/azure-sdk-bom/README.md

```diff
@@ -29,16 +29,7 @@ result in all dependencies being included in your project.
 </dependencyManagement>
 ```
 
-#### Determine the version of azure-sdk-bom according to the version of Spring Boot
-
-If you are using Spring Boot in your project, you can determine the version of azure-sdk-bom according to the version of Spring Boot.
-
-| Version of Spring Boot | Version of azure-sdk-bom |
-| ---------------------- | ----------------------- |
-| 2.6.x                  | 1.1.0                   |
-| 2.5.x                  | 1.0.6                   |
-| 2.4.x                  | 1.0.3                   |
-| < 2.4.0                | Not supported           |
+**NOTE**: In **Spring Boot** application, you can choose **azure-sdk-bom**'s version according to [Spring-Versions-Mapping](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Versions-Mapping).
```

Comment:
let's use the `https://aka.ms/spring/versions` link here.

Reply:
Updated.